README.md

```diff
@@ -17,6 +19,8 @@ Alternatively, if you would like to edit the files locally:
 git clone https://github.com/nf-core/tools.git nf-core-tools
 cd nf-core-tools
 python setup.py develop
+# Or with pip
+pip install -e .
```

Comment:
Nice, didn't know the `-e` flag for pip. What's the difference between this command and the one above?

Reply:
Me neither :) I just did some research. Some people say that the dependency resolving will not working right in all cases with setup.py in a local environment. Also, using pip enables you to easily uninstall the local package again.